GeeksCoreLibrary/Modules/PostalServices/NeDistri/Models/LabelRule.cs

```diff
@@ -0,0 +1,7 @@
+namespace GeeksCoreLibrary.Modules.PostalServices.NeDistri.Models;
+
+public class LabelRule
```

Comment:
Can you clarify the properties of this class with summaries? Currently without knowing NeDistri it is unclear what these properties contain.